runtime/doc/api.txt

```diff
@@ -1590,7 +1610,13 @@ nvim_set_hl({ns_id}, {name}, {*val})                           *nvim_set_hl()*
                                |highlight-ctermbg|
                              • cterm: cterm attribute map, like
                                |highlight-args|. Note: Attributes default to
-                               those set for `gui` if not set.
```

Comment:
The `gui` term here is a bit problematic, should it instead say "inherit the formatting rules from the booleans above" or something? We don't really define what `gui` is, or what "formatting rules" are either.

Reply:
Perhaps

> Note: if not set, cterm attributes will match those from the attribute map documented above.

I don't *think* you need to point out that fg/bg wont be copied as those are explicitly listed as ctermfg/bg. I did call it "primary attribute map above" but I think primary is superfluous.

Not sure if you need "supported cterm attributes", I think only `blend` isn't supported in cterm? If so, probably I'd just tack "Blend is not supported in cterm mode(?)" on the end to remove ambiguity.